### ISSUE TITLE
4.2.0.-beta1 possible bug: Config class redlecaration 

### ISSUE THREAD

**Issue by ghost** [contao/installation-bundle#13]
<a href="https://github.com/DanielSchwiperich"><img src="https://avatars.githubusercontent.com/u/2045722?v=3" align="left" width="42" height="42"></img></a> [Issue](https://github.com/contao/core-bundle/issues/476) by @DanielSchwiperich
April 25th, 2016, 19:07 GMT

Wanted to test #426 in a fresh install and used the new beta release. However when calling the install.php I get the following error: 

```
Fatal error: Cannot redeclare class Contao\Config in /var/www/share/contao-426.test.local/htdocs/standard-edition-4.2.0-beta1/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Config.php on line 22
```

Does not happen with 4.1.3. 
Did not dig into it. Maybe s1 can confirm (or not)?

**Comment by ghost** [comment]
<a href="https://github.com/leofeyer"><img src="https://avatars.githubusercontent.com/u/1192057?v=3" align="left" width="42" height="42"></img></a> Comment by @leofeyer
April 26th, 2016, 08:04 GMT

How did you install it? `composer create-project`?

**Comment by ghost** [comment]
<a href="https://github.com/DanielSchwiperich"><img src="https://avatars.githubusercontent.com/u/2045722?v=3" align="left" width="42" height="42"></img></a> Comment by @DanielSchwiperich
April 26th, 2016, 08:18 GMT

No just downloaded the beta release and then did a composer install. 
Should have mentioned that I used the standard-edition, but posted the error here since it was caused in a core bundle file. 

https://github.com/contao/standard-edition/releases/tag/4.2.0-beta1

**Comment by leofeyer** [comment]
Fixed in 76ca55efa67bc4a385fbdeb3e084eb511c8086f7.
